imod/mf6/out/cbc.py

```diff
@@ -288,8 +297,9 @@ def read_imeth6_budgets_dense(
     Three-dimensional array of floats
     """
     # Allocates a dense array for the entire domain
-    out = np.zeros(size, dtype=np.float64)
+    out = np.full(size, np.nan, dtype=np.float64)
```

Comment:
Using NaN instead of zeros is an okay choice, but it should be mentioned in the changelog.

I think I used zeros here because iMODFLOW, iMOD-WQ do the same.
It makes adding different water balance terms a bit easier. Maybe open a separate issue for this?